Course/Course/Presentation/Unit/CourseUnitViewModel.swift

```diff
@@ -24,11 +24,11 @@ public enum LessonType: Equatable {
         case .discussion:
             return .discussion(block.topicId ?? "", block.id, block.displayName)
         case .video:
-            if block.youTubeUrl != nil, let encodedVideo = block.videoUrl {
+            if block.youtubeVideoUrl != nil, let encodedVideo = block.video(streamingQuality: streamingQuality)?.url {
```

Comment:
I'm bit confused here, we are checking for youtube video and then returning the same data that we are returning in the case of native videos. Am I missing something there?

Reply:
This is old logic and not apply to task